### PR TITLE
Dev mani

### PR DESCRIPTION
**Pull request by Manikandan610**
Refector using map and short the code

### CODE REVIEW

File: src/Components/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+type contentDetailsProps = {
+    id: any
+    title: any
+    subtitle: any
+    description: string
+}
+
+const Content = (props:contentDetailsProps) => {
+  return (
+
```
Comment (buda-fibonalabs) [inline src/Components/Content.tsx]: remove unnecessary spaces here! 
follow one convention, I think here it is 2 spaces everywhere

File: src/Components/Content.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    
+                <Col className="gutter-row" span={4}>
+                    <div className='cols'>
+                        <AreaChartOutlined className='iconstyle'/>
+                        <div className='col_content'>
+                                <h1>{props.title}<sub>{props.subtitle}</sub></h1>
+                            </div>
+                        <p>{props.description}</p>
+                    </div>
+                </Col>
```
Comment (buda-fibonalabs) [inline src/Components/Content.tsx]:
```suggestion
              <Col className="gutter-row" span={4}>
                  <div className='cols'>
                      <AreaChartOutlined className='iconstyle'/>
                        <div className='col_content'>
                              <h1>{props.title}<sub>{props.subtitle}</sub></h1>
                         </div>
                        <p>{props.description}</p>
                  </div>
              </Col>
```

**Review by buda-fibonalabs — CHANGES_REQUESTED** [review]

File: src/Components/Content.tsx (hunk 1 above)
Comment (buda-fibonalabs) [reply src/Components/Content.tsx]: just run format with preetier
i will set up a common .preetierrc soon

File: src/Components/ModalComp.tsx [hunk 3]
```diff
@@ -24,106 +21,19 @@ const ModalComp = () => {
...
-                    </div>
-                </Col>
-                <Col className="gutter-row" span={4}>
-                    <div className='cols'>
-                        <AreaChartOutlined className='iconstyle'/>
-                        <h1>24:00</h1>
-                        <p>Hours of Charge</p>
-                    </div>
-                </Col>
+                {contentDetails.map((contentDisp:any) => {
```
Comment (buda-fibonalabs) [inline src/Components/ModalComp.tsx]: Nice change ! 👍

File: src/Components/Content.tsx (hunk 1 above)
Comment (Manikandan610) [reply src/Components/Content.tsx]: I removed unwanted spaces.

**Review by buda-fibonalabs — APPROVED** [review]

**Pull request merged**